src/vs/workbench/api/common/extHostLanguageFeatures.ts

```diff
@@ -2542,7 +2542,9 @@ export class ExtHostLanguageFeatures implements extHostProtocol.ExtHostLanguageF
 			onEnterRules: configuration.onEnterRules ? ExtHostLanguageFeatures._serializeOnEnterRules(configuration.onEnterRules) : undefined,
 			__electricCharacterSupport: configuration.__electricCharacterSupport,
 			__characterPairSupport: configuration.__characterPairSupport,
+			autoClosingPairs: configuration.autoClosingPairs
```

Comment:
@alexdima, should there be a `checkProposedApiEnabled` somewhere here?

Reply:
Good catch! I've updated the PR.